trufflehog-exclude-patterns.txt

```diff
@@ -0,0 +1,2 @@
+(.*/)?test/(.*/)?
+(.*/)?htmlcov/(.*/)?
```

Comment:
With this file we use this patterns to ignore mock secret key matches in our test and coverage files. Ideally though we should not hardcode those keys in the tests.